test/test_helper.rb

```diff
@@ -1,11 +1,7 @@
 # frozen_string_literal: true
 
-require 'rails/all'
-require 'aws_sdk_rails'
 require 'minitest/autorun'
 
-ENV['RAILS_ENV'] = 'test'
```

Comment:
Note that although this line is not required for our current purposes, it will be required if any `ActionDispatch::IntegrationTest` test classes are added in the future.  (See jonathanhefner/garden_variety@55af1a2 for an explanation.)

Reply:
Ah ok; added back, then.